hooks/playbooks/adoption_deploy_ceph.yml

```diff
@@ -143,6 +143,18 @@
             state: present
           loop: "{{ _tripleo_nodes }}"
 
+        # NOTE: TripleO has the hardcoded --yes-i-know option that is not valid anymore
+        # in RHCS 7. TripleO does not receive any new patch both upstream and downstream
+        # (it is a retired project), hence the only option we have is to patch the
+        # current code to not have that line.
+        - name: Patch Ceph bootstrap task
```

Comment:
@marios FYI as done for `install_yamls`, we need to patch the bootstrap command here as well.

Reply:
it seems so much more civilised in ansible ;)